style.css

```diff
@@ -271,6 +348,118 @@ section.headline .social-media li a i.fa-github {
   }
 }
 
+/* Start of modal */
+
+.modal {
+  /* display: flex; */
+  flex-direction: column;
+  position: fixed;
+  justify-content: center;
+  align-content: center;
+  height: 100vh;
+  right: 0;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  z-index: 200;
+  backdrop-filter: blur(15px);
+  padding: 16px;
+  margin: 16px;
+  background-color: #fff;
+  overflow-y: auto;
+}
+
+.modal-content {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+  margin-bottom: 35px;
+}
+
+.modal-content .modal-heading {
+  /* font-family: 'Roboto', sans-serif; */
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 35px;
+}
+
+.closeBtn {
+  color: #ccc;
+  font-size: 30px;
+  margin-top: 5px;
+}
+
+.closeBtn:hover,
+.closeBtn:focus {
+  color: black;
+  cursor: pointer;
+}
+
+.modal-describ {
+  /* font-family: 'Inter', sans-serif; */
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 20px;
+  color: #344563;
+  margin-top: 10px;
+}
+
+#modalList {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 35px;
+}
+
+#modalList .list-skills {
+  /* font-family: 'Poppins', sans-serif; */
+  font-style: normal;
+  font-weight: 100;
+  font-size: 13px;
+  line-height: 16px;
+  border: 1px solid silver;
+  text-align: center;
+  padding: 5px;
+  margin: 0 0 16px;
+}
+
+.modalImg {
+  margin: 0 08px;
+}
+
+/* .double-btn {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  color: #fff;
+  margin: 8px 0 10px;
+}
+
+.double-btn a {
+  font-family: Inter sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
```

Comment:
* Kindly remove all commented code

Reply:
I appreciate your comment over this, and I do understand your point.
But this is so, because am still working on my modal. Kindly ignore that as it will be taking care of once my modal is finalized.

Thanks